api/src/main/kotlin/org/jetbrains/kotlinx/dl/api/core/layer/pooling/GlobalAvgPool1D.kt

```diff
@@ -14,26 +14,27 @@ import org.tensorflow.op.Ops
 
 /**
  * Global average pooling operation for temporal data.
- * NOTE: Works with tensors which must have rank 3 (batch, steps, features).
- * Input shape: 3D tensor with shape `(batch_size, steps, features)`.
- * Output shape: 2D tensor with shape `(batch_size, features)`.
- * @property [name] Custom layer name.
+ *
+ * NOTE: Only works with tensors which have rank 3, i.e. tensors with shape
+ * `(batch, steps, features)`.
```

Comment:
Sorry, why did you remove the information about the outputSHape

Reply:
Well, the input shape is already included in the "NOTE". And probably I have removed the output shape just due to consistency with documentation of other pooling layers. I'll revert back this change if you want.

Reply:
Yes, please remove this documentation, I'm going to add this information for all layers, think it will be useful for the user.